run_udf.json

```diff
@@ -93,7 +93,7 @@
         }
     },
     "returns": {
-        "description": "The data processed by the UDF. Returns a raster data cube if a raster data cube was passed for `data`. If an array was passed for `data`, the returned value is defined by the context and is exactly what the UDF returned.",
+        "description": "The data processed by the UDF.\n\n* Returns a raster data cube, if a raster data cube was passed for `data`. Details on the dimensions and dimension properties (name, type, labels, reference system and resolution) depend on the UDF.\n* If an array was passed for `data`, the returned value can be of any data type, but is exactly what the UDF returned.",
```

Comment:
>  was passed for `data`

is passed for `data`

> If an array was passed for `data`, the returned value can be of any data type, but is exactly what the UDF returned.

If an array is passed for `data`, the returned value can be of any data type, but is exactly what the UDF returns.